models/eprop_iaf.cpp

```diff
@@ -214,6 +229,18 @@ eprop_iaf::Parameters_::set( const DictionaryDatum& d, Node* node )
     throw BadProperty( "Cutoff of integration of eprop trace between spikes eprop_isi_trace_cutoff ≥ 0 required." );
   }
 
+  if ( delay_rec_out_ < 1 )
+  {
+    throw BadProperty( "Connection delay from recurrent to output neuron ≥ 1 required." );
+  }
```

Comment:
In cpp we refer to the output neuron as readout neuron. I know that there is an inconsistency to the python examples where we refer to them as output neurons, but we should for now be consistent on the cpp level and python level, individually. A todo with low priority is to change the cpp layer to match the python level.
```suggestion
  if ( delay_rec_out_ < 1 )
  {
    throw BadProperty( "Connection delay from recurrent to readout neuron ≥ 1 required." );
  }
```